src/AccessibilityInsights.SharedUx/Misc/VersionTools.cs

```diff
@@ -11,8 +14,10 @@ public static class VersionTools
         /// <returns>The file version string (as formatted in the file resources)</returns>
         public static string GetAppVersion()
         {
-            string fileVersion = System.Diagnostics.FileVersionInfo.GetVersionInfo(System.Reflection.Assembly.GetExecutingAssembly().Location).FileVersion;
+            string fileVersion = FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly().Location).FileVersion;
             return fileVersion;
         }
+
+        public static string GetAxeVersion() => FileVersionInfo.GetVersionInfo(Assembly.GetAssembly(typeof(CaptureAction)).Location).FileVersion;
```

Comment:
>FileVersion [](start = 133, length = 11)

I thought Rob had added a way to get the exact string that shows up in Nuget. I'd much prefer to use that than trying to reverse engineer it. In [https://www.nuget.org/packages/Axe.Windows](https://www.nuget.org/packages/Axe.Windows), the version is 0.3.4-prerelease.

Reply:
Yes, `ProductVersion` looks like a much better approach. I added v because it is in the mock but I am happy to remove it if we don't really want it

Reply:
Updated